database/README.md

```diff
@@ -37,4 +38,10 @@ Fake subscriber and article data can be generated for Fox News. This is useful f
 The mapping between article headlines and topics can be used for validation later on in the pipeline.
 
 1. Reset the database
-2. Run `python3 -m generate_dummy_data.py` to seed the database with dummy data.
\ No newline at end of file
+2. Run `python3 -m generate_dummy_data.py` to seed the database with dummy data.
+
+### Explanation of political leaning attribute in schema
+
+Within the `source` table of the database schema, there is a column entitled `source_political_leaning`. We thought it important to include this information about the political orientation of a source in the database, so that users of the finished project (including the dashboard) can filter for sources that sit at particular positions on the political spectrum. 
+
+However, politics as a field is not strictly defined, and there is no singular correct way of classifying the political leaning of a newspaper, politician, or article. We decided to use the [Interactive Media Bias Chart](https://adfontesmedia.com/interactive-media-bias-chart/) to classify the political leanings of the newspapers we have chosen for this project. We found this to be most useful as it not only classifies news sources into 'left' and 'right', but also includes information on what sort of content a news source publishes - from "Original Fact Reporting, High Effort", to "Contains Inaccurate / Fabricated Info". We tried to pick sources from opposite ends of the political spectrum, but also sources that published generally reliable content, and we felt that Fox News and Democracy Now! were the best fit. Ad Fontes Media (the creators of the Interactive Media Bias Chart) include more information on how they created the chart and how they did so to mitigate bias as much as possible, which can be found at [this link](https://adfontesmedia.com/is-the-media-bias-chart-biased/).
```

Comment:
Nice - really interesting and also includes links for those who want to know more. Clear language too for those who don't have a politics background.